src/app/common/swaps/hooks.ts

```diff
@@ -0,0 +1,5 @@
+export function useSwapFeature() {
+  return {
+    isSwapEnabled: !!process.env.SWAP_ENABLED
```

Comment:
I removed this, it is now handled in our `environment.ts` file ...no need for the hook. Is this rebased?

Reply:
@fbwoolf it hasn't been rebased because the order still had to be defined as of our last sync. Should I rebase this onto #4205? How much is that branch still changing to allow rebasing and keep working on my end without encountering too many conflicts later?